synapse/handlers/federation_event.py

```diff
@@ -1476,38 +1476,35 @@ async def backfill_event_id(
         Raises:
             FederationError if we are unable to find the event from the destination
         """
-        logger.info(
-            "backfill_event_id: event_id=%s from destination=%s", event_id, destination
-        )
+        logger.info("backfill_event_id: event_id=%s", event_id)
 
         room_version = await self._store.get_room_version(room_id)
 
-        event_from_response = await self._federation_client.get_pdu(
-            [destination],
+        pulled_pdu_info = await self._federation_client.get_pdu(
+            destinations,
             event_id,
             room_version,
         )
 
-        if not event_from_response:
+        if not pulled_pdu_info:
             raise FederationError(
                 "ERROR",
                 404,
-                "Unable to find event_id=%s from destination=%s to backfill."
-                % (event_id, destination),
```

Comment:
TODO: We can put the `destination` back since we can access it via `pulled_pdu_info.pull_origin`

Reply:
Actually, this is the case where there is no `pulled_pdu_info` to look at. Disregard